### DISCUSSION TITLE
Some UI Kit components not migrated?

### DISCUSSION THREAD

**Discussion by bengalley-e2x** [commercetools/ui-kit#2144] [Q&A] [answered]
In the legacy documentation, there are some UI components that are described separately to rest of the UI Kit. Such as `ConfirmationDialog` (which I've used extensively in a custom app).

<img width="174" alt="Screenshot 2022-03-08 at 10 34 26" src="https://user-images.githubusercontent.com/80111772/157219588-9709a48d-1a6d-46ca-bf01-44634a89c5f4.png">

Are these also being migrated or are they being deprecated?

Are they documented anywhere currently?

Thanks,
Ben

**Comment by emmenko** [comment, marked as answer]
Hi, thanks for asking.

We changed a bit the documentation structure yes, as some parts of the (legacy) documentation were a bit out of place and at times confusing.

These components now are documented in their respective `API Reference` section. For instance, the dialog and modal pages components are documented in the `@/application-components` page.
https://docs.commercetools.com/custom-applications/api-reference/commercetools-frontend-application-components

Nothing has been deprecated about these components btw.

Let us know if you have further questions. Thanks
